File: fixtures/scrapy_output/get_user_agent.py
from scrapy import Request, Spider


class MySpider(Spider):
    name = 'myspider'

    def start_requests(self):
        headers = {
            'x-msisdn': 'XXXXXXXXXXXXX',
            'User-Agent': 'Mozilla Android6.1',
        }
        yield Request(
            'http://205.147.98.6/vc/moviesmagic?p=5&pub=testmovie&tkn=817263812',
            callback=self.parse,
            headers=headers,
        )

    def parse(self, response):
        pass

File: fixtures/elixir_output/get_user_agent.ex
request = %HTTPoison.Request{
  method: :get,
  url: "http://205.147.98.6/vc/moviesmagic",
  options: [],
  headers: [
    {~s|x-msisdn|, ~s|XXXXXXXXXXXXX|},
    {~s|User-Agent|, ~s|Mozilla Android6.1|},
  ],
  params: [
    {~s|p|, ~s|5|},
    {~s|pub|, ~s|testmovie|},
    {~s|tkn|, ~s|817263812|},
  ],
  body: ""
}

response = HTTPoison.request(request)
